app/components/bucket-page/bucket-page.coffee

```diff
@@ -41,4 +41,7 @@ module.exports =
     $scope.userCanManageBucket = ->
       $scope.bucket && !$scope.bucket.isArchived() && ($scope.membership.isAdmin || $scope.bucket.author().id == $scope.membership.member().id)
 
+    $scope.userCanUnarchiveBucket = ->
+      $scope.bucket && ($scope.membership.isAdmin || $scope.bucket.author().id == $scope.membership.member().id)
+
```

Comment:
i can't see where else this is happening or _if_ it's happening, but a user can only unarchive a bucket if it is fully funded.